Shopify/cli
# Add handle persistence to app deploy mutations

### WHY are these changes introduced?

Part of: https://github.com/Shopify/develop-app-foundations/issues/26
Requires: https://github.com/Shopify/shopify/pull/451765, https://github.com/Shopify/partners/pull/50186, https://github.com/Shopify/partners/pull/50232

This PR is being added to persist `handle` from the CLI so that any `shopify.extension.toml` that specifies `handle` will send it up to Core so that we can persist it server-side.

### WHAT is this pull request doing?



### How to test your changes?

1. Use branches in the aforementioned PRs
2. Set up `spin` to create an app with the CLI and generate an Admin Action extension 
3. Run `shopify app deploy` for your app to deploy and release the changes
4. Verify in the core DB that the `handle` is persisted for an extension version

### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
